Client/mods/deathmatch/logic/luadefs/CLuaWorldDefs.cpp

```diff
@@ -131,7 +132,8 @@ void CLuaWorldDefs::LoadFunctions()
                                                                              {"isPedTargetingMarkerEnabled", IsPedTargetingMarkerEnabled},
                                                                              {"isLineOfSightClear", IsLineOfSightClear},
                                                                              {"isWorldSpecialPropertyEnabled", ArgumentParserWarn<false, IsWorldSpecialPropertyEnabled>},
-                                                                             {"isGarageOpen", IsGarageOpen}};
+                                                                             {"isGarageOpen", IsGarageOpen},
+                                                                             {"isVolumetricShadowsEnabled", ArgumentParserWarn<false, IsVolumetricShadowsEnabled>}};
```

Comment:
This function didn't exist before, so there's no reason to use `ArgumentParserWarn`, please just use `ArgumentParser` here.